pkg/routeagent_driver/handlers/kubeproxy_iptables/constants.go

```diff
@@ -32,7 +32,7 @@ const (
 	// To support connectivity for Pods with HostNetworking on the GatewayNode, we program
```

Comment:
Should this comment move with the variable?

Reply:
Oh it should! good catch!

Reply:
I will address this on separate PR if that's ok.